doc/lspconfig.txt

```diff
@@ -362,11 +365,12 @@ The three steps for adding and enabling a new server configuration are:
    lspconfig.foo_lsp.setup{}
 <
 ==============================================================================
-INDEX                                                          *lspconfig-index*
+CONFIGURATIONS                                 *lspconfig-server-configurations*
```

Comment:
won't the tag from this conflict with the tag from the tagfile? Should we just delete this section?